frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.unit.spec.js

```diff
@@ -60,7 +60,7 @@ function setup({ items, collection } = {}) {
 }
 
 describe("PinnedItemOverview", () => {
-  it("should render an empty banner when there are no items", () => {
+  it.skip("should render an empty banner when there are no items", () => {
```

Comment:
I'm assuming this banner will come back eventually, so just skipping this test for now.